packages/eas-cli/src/commandUtils/context/__tests__/ProjectDirContextField-test.ts

```diff
@@ -56,7 +40,7 @@ describe(ProjectDirContextField['findProjectRootAsync'], () => {
       },
       '/app'
     );
-    const projectRoot = await ProjectDirContextField['findProjectRootAsync']({ cwd: '/app/src' });
+    const projectRoot = await ProjectDirContextField['findProjectRootAsync']();
```

Comment:
The point of this test was to check if `process.cwd` is in the subdirectory will the function return directory with package.json. If you mock the entire pkg-dir then this test does not check anything at this point.